content/news/XXXX-XX-XX-mixxx-2-5-beta-released.md

```diff
@@ -0,0 +1,493 @@
+title: Mixxx 2.5 beta released
+authors: Evelynne Veys
+tags: 2.5, beta, release announcement
+comments: yes
+status: draft
+
+#### Dear Mixxx-ers
+
+Funny things happen in spring, birds start nesting and lay eggs, and so did the Mixxx developers.
+A bit late for Easter but it is a magnificent giant gift filled with smaller gifts: Mixxx 2.5 beta!
+You can find the details about the new version below.
+
+#### Appeal
+
+Before releasing 2.5 as the next stable release we need it to be tested thoroughly, something we can't do on our own.
+After some time looking at the same release and searching for solutions we might forget miner bugs or problems.
+That's why we need **YOU** to help us. The more people testing this Beta and reporting eventual bugs to the [bugtracker](https://github.com/mixxxdj/mixxx/issues), the faster the beta can be promoted to stable.
+In case you're scared of messing up your neetly configured PC and losing your holy data is your Haloween-nightmare, read the new wiki article [safeguard your Mixxx Data](https://github.com/mixxxdj/mixxx/wiki/Safeguard%20your%20Digital%20DJ%20Data).
```

Comment:
new url is
https://github.com/mixxxdj/mixxx/wiki/Safeguard-your-Digital-DJ-Data

Reply:
fixed